src/uk/co/lloydsbanking/kotlin/hours/stdlib/Let.kt

```diff
@@ -0,0 +1,18 @@
+package uk.co.lloydsbanking.kotlin.hours.stdlib
```

Comment:
I think some people might be happier if instead of the name of our employer we refer to us as `llobile` :)